mix.exs

```diff
@@ -40,16 +40,16 @@ defmodule Membrane.AudioMix.Mixfile do
 
   defp deps do
     [
-      {:membrane_core, "~> 0.7.0"},
-      {:membrane_caps_audio_raw, "~> 0.4.0"},
-      {:unifex, "~> 0.6.0"},
-      {:bunch, "~> 1.0"},
-      {:ex_doc, "~> 0.24", only: :dev, runtime: false},
-      {:dialyxir, "~> 1.0.0", only: :dev, runtime: false},
-      {:credo, "~> 1.5", only: :dev, runtime: false},
-      {:membrane_file_plugin, "~> 0.6.0", only: :test},
-      {:membrane_common_c, "~> 0.8.0"},
-      {:membrane_mp3_mad_plugin, "~> 0.8.0", only: :test}
+      {:membrane_core, "~> 0.9.0"},
+      {:membrane_raw_audio_format, "~> 0.8.0"},
```

Comment:
Migration to `raw_audio_format` was already done in `v0.7.0`, change here is caused by merging `master` into `native-mixer`

Reply:
OK, so for a merge to master this should be consistent and use `raw_audio_format`, not caps.
The trouble is it won't be usable in Dubber without updating all elements to use `raw_audio_format`. Fortunately, it seems all the required elements have been updated while this PR was open, so we won't have to create a branch using caps.

One more thing though - I'd like to set a deadline for changes to this PR, so in case the renaming wasn't ready until tomorrow, please let me know